Sources/SQL/Data/DataQuery.swift

```diff
@@ -74,6 +77,19 @@ public enum OrderByDirection {
     case descending
 }
 
+public struct DataGroupBy {
+    public var column: DataColumn?
+    public var custom: String?
```

Comment:
I think this might make more sense as enum enum with two cases

Reply:
agreed

Reply:
done